spec/bowlingSpec.js

```diff
@@ -0,0 +1,97 @@
+describe('ScoreCard', function() {
+  var scoreCard;
+
+  beforeEach(function() {
+    scoreCard = new ScoreCard();
+  });
+
+  describe('scoreCard', function() {
+    it('displays scores as zero at the start', function() {
+      expect(scoreCard.scoreTotal).toBe(0);
+    });
+  });
+
+  describe('count', function() {
+    it('stores the number of fallen pins in a roll', function() {
+      scoreCard.roll(6);
+      expect(scoreCard.rollArray[0]).toBe(6);
+    });
+  });
+
+  describe('frameScores', function() {
+    it('adds up the number of fallen pins in two rolls', function() {
+      scoreCard.roll(6);
+      scoreCard.roll(3);
+      expect(scoreCard.frameScores[0]).toBe(9);
+    });
+  });
+
+  describe('strike', function() {
+    it('ends the frame', function() {
+      scoreCard.roll(10);
+      scoreCard.roll(2);
+      expect(scoreCard.rollArray[1]).toBe(0);
+    });
+  });
+
+  describe('strike bonus', function() {
+    it('adds next two rolls as bonus points', function() {
+      scoreCard.roll(10);
```

Comment:
Have a helper method that will do all this rolling for you. Then you can do:
```ruby
manyRolls(10, 0, 3, 4,)
```
For example.